### ISSUE TITLE
Dask dashboards to not work in teehr-hub

### ISSUE THREAD

**Issue by mgdenno** [RTIInternational/teehr#62]
Not sure of the extent of this issue, but it seems that there is definitely an issue with the dashboard plots in teehr-hub.  They just don't load. This may be related to #61 

@samlamont @kvanwerkhoven tracking this issue here.

**Comment by mgdenno** [comment]
To recreate:
```
from dask.distributed import Client
client = Client()
client
```
try to load dashboard in side bar to the left, and open any plot.

**Comment by mgdenno** [comment]
Verified resolved in v0.3.10
